### ISSUE TITLE
Feature Request: Notification to be sent to the listing owner via email when a listing is about to expire.

### ISSUE THREAD

**Issue by adampickering** [mikejolley/WP-Job-Manager#446]
Options along the lines of these:

![img](https://dl.dropboxusercontent.com/spa/nnc1royxnaf5onn/cl-ykor2.jpg)

**Comment by adampickering** [comment]
Would be also great to email the site admin as well as the listing owner. Perhaps an option to enable this?

**Comment by kochhase** [comment]
this feature would be nice to have! And is there a plan to give this jobowner the posibility to have a discount for the next jobpage?

**Comment by Roy1980** [comment]
I need this badly!

**Comment by adampickering** [comment]
:+1:

**Comment by Designerplus** [comment]
nice one!

**Comment by Roy1980** [comment]
Any news about this?

**Comment by mikejolley** [comment]
https://wordpress.org/plugins/post-status-notifier-lite/ can be used to end an email once expired.  No short term plans to add this feature request to core.

**Comment by kochhase** [comment]
sorry Mike,
I talk to Timo Reith, he is gernan guy. I use his post status notifier pro 1.8. But how should he do this job? He can do this only when the jobstatus changed. Or is he able to look in the time field to evaluate the periods? I don´t now how.

**Comment by timoreith** [comment]
That's right, my plugin (Post Status Notifier, link above posted by Mike) only triggers on post status changes. I could imagine a cronjob that walks through all job listings and checks the expiration date. When it encounters a match, like four days before expiration from the screenshot, it updates the listing and adds a category "expiring". Then you could create a PSN notification rule that triggers for posts having this category.
That's just my first idea. I guess we could find a better solution.

**Comment by kochhase** [comment]
Hi Timo, that sounds good. Do you think, we could than send a link to spezific Jobpackets to give the  jobowner the posibility to have a discount for the next jobpacket?

**Comment by kochhase** [comment]
Hi Timo,
do you know, when we can aspect some trigger for this problem? I need that feature for my update of meerjobs. thank you for help!!

**Comment by craigg5** [comment]
I would really like to see this feature added also!

**Comment by tripflex** [comment]
This is a feature available in the emails addon FYI https://plugins.smyl.es/wp-job-manager-emails

**Comment by adampickering** [comment]
Still I think it really should be a core feature

**Comment by annezazu** [comment]
599924-zen
